src/ledger/ledger.js

```diff
@@ -245,30 +287,62 @@ export class Ledger {
       name: existingUser.name,
       aliases: updatedAliases,
     };
+    const addr = userAddress(userId);
+    const aliasAccount = this._unsafeGetAccount(alias);
+    const userAccount = this._unsafeGetAccount(addr);
 
     // State mutations! Method must not fail past this comment.
     this._users.set(userId, updatedUser);
+    // Transfer alias's balance and paid to the canonical account
+    userAccount.paid = G.add(userAccount.paid, aliasAccount.paid);
+    userAccount.balance = G.add(userAccount.balance, aliasAccount.balance);
```

Comment:
I have some concerns about not logging these transfers in the ledger. It'd definitely make returning maliciously captured funds through fraudulent alias accumulation more straightforward, since we can refer to a specific ledger entry, in addition to any ill-gotten grain through proportional cred accumulation